### DISCUSSION TITLE
I cannot apply for education discounts because of a bug

### DISCUSSION THREAD

**Discussion by 134820134820** [community/community#77524] [GitHub Education]
### Hi everyone,

First of all, let's talk about the problem. My phone cannot call the camera for shooting. On my iPhone, I can only call one front facing camera, but this is too difficult to shoot, and the other phone even cannot call anyone
Firstly, on my computer, when I first selected my email and applied, I clicked the application button and there was no translation available. Upon checking, I found that there seemed to be no requests sent either
Then I tried to apply using a mobile browser, but both Safari and Via were unable to pop up requests when address authentication was required, preventing me from proceeding to the next step
Finally, when I entered the evidence submission interface using another browser, I found that it could only call the front camera of my iPhone?? Choosing a different camera prevents me from displaying images or taking photos for uploading. Additionally, my phone seems to be unable to upload images locally, causing me to get stuck (When I tried using another phone, I couldn't use any camera directly, and clicking to upload pictures didn't respond.) I think this should be a bug

**Comment by hackelite01** [comment]
hey @134820134820 , I can do it easily. Drop the details on mail at [hackelite.sup@gmail.com](mailto:hackelite.sup@gmail.com)
